riscv/cachesim.h

```diff
@@ -100,6 +100,10 @@ class cache_memtracer_t : public memtracer_t
   {
     cache->set_log(log);
   }
+  void print_stats(void)
```

Comment:
No need for the `(void)`; empty parentheses suffice

Reply:
Thank you for your feedback. I will submit patches with corresponding changes shortly.